packages/launchpad/cypress/e2e/project-setup.cy.ts

```diff
@@ -584,7 +584,7 @@ describe('Launchpad: Setup Project', () => {
     // TODO: Had to revert due to regression: https://github.com/cypress-io/cypress/pull/26452
     // Would be great to fully support Plug n Play eventually, but right now it causes issues relating
     // to not correctly detecting dependencies when installing the binary.
-    it.skip('works with Yarn 3 Plug n Play', () => {
+    it.only('works with Yarn 3 Plug n Play', () => {
```

Comment:
I have tried reworking this test to feature an angular repo that should be spun up via cypress. I have run into some other pnp-related issues, that could be mitigated, but i'm stuck at an esm import error deep in the `angular_devkit/build` package
```
 cypress:lifecycle:ProjectConfigIpc plugins process error: Error [ERR_MODULE_NOT_FOUND]: Cannot find package '@angular/compiler-cli' imported from /private/var/folders/hg/3s1q3w8j4xlgbp55q9cbsnkm0000gn/T/cy-projects/yarn-v3.1.1-pnp/.yarn/__virtual__/@angular-devkit-build-angular-virtual-4aa3a8673f/0/cache/@angular-devkit-build-angular-npm-16.2.7-ff0a3596ac-cfd587b125.zip/node_modules/@angular-devkit/build-angular/src/utils/load-esm.js
[cy:open:dev:30509]: Did you mean to import @angular-compiler-cli-virtual-8bb163dda9/0/cache/@angular-compiler-cli-npm-16.2.10-071c53740f-07c89fc8a0.zip/node_modules/@angular/compiler-cli/bundles/index.js?
[cy:open:dev:30509]:     at new NodeError (node:internal/errors:399:5)
[cy:open:dev:30509]:     at packageResolve (node:internal/modules/esm/resolve:895:9)
[cy:open:dev:30509]:     at moduleResolve (node:internal/modules/esm/resolve:944:20)
[cy:open:dev:30509]:     at defaultResolve (node:internal/modules/esm/resolve:1159:11)
[cy:open:dev:30509]:     at nextResolve (node:internal/modules/esm/loader:163:28)
[cy:open:dev:30509]:     at ESMLoader.resolve (node:internal/modules/esm/loader:838:30)
[cy:open:dev:30509]:     at ESMLoader.getModuleJob (node:internal/modules/esm/loader:424:18)
[cy:open:dev:30509]:     at ESMLoader.import (node:internal/modules/esm/loader:525:22)
[cy:open:dev:30509]:     at importModuleDynamically (node:internal/modules/cjs/loader:1193:29)
[cy:open:dev:30509]:     at importModuleDynamicallyWrapper (node:internal/vm/module:429:21)
[cy:open:dev:30509]:     at importModuleDynamically (node:internal/vm:106:46)
[cy:open:dev:30509]:     at importModuleDynamicallyCallback (node:internal/process/esm_loader:35:14)
[cy:open:dev:30509]:     at eval (eval at loadEsmModule (/private/var/folders/hg/3s1q3w8j4xlgbp55q9cbsnkm0000gn/T/cy-projects/yarn-v3.1.1-pnp/.yarn/__virtual__/@angular-devkit-build-angular-virtual-4aa3a8673f/packages/angular_devkit/build_angular/src/utils/load-esm.ts:24:10), <anonymous>:3:1)
[cy:open:dev:30509]:     at loadEsmModule (/private/var/folders/hg/3s1q3w8j4xlgbp55q9cbsnkm0000gn/T/cy-projects/yarn-v3.1.1-pnp/.yarn/__virtual__/@angular-devkit-build-angular-virtual-4aa3a8673f/packages/angular_devkit/build_angular/src/utils/load-esm.ts:24:66)
[cy:open:dev:30509]:     at readTsconfig (/private/var/folders/hg/3s1q3w8j4xlgbp55q9cbsnkm0000gn/T/cy-projects/yarn-v3.1.1-pnp/.yarn/__virtual__/@angular-devkit-build-angular-virtual-4aa3a8673f/packages/angular_devkit/build_angular/src/utils/read-tsconfig.ts:29:71)
[cy:open:dev:30509]:     at generateWebpackConfig (/private/var/folders/hg/3s1q3w8j4xlgbp55q9cbsnkm0000gn/T/cy-projects/yarn-v3.1.1-pnp/.yarn/__virtual__/@angular-devkit-build-angular-virtual-4aa3a8673f/packages/angular_devkit/build_angular/src/utils/webpack-browser-config.ts:46:38)
[cy:open:dev:30509]:     at generateBrowserWebpackConfigFromContext (/private/var/folders/hg/3s1q3w8j4xlgbp55q9cbsnkm0000gn/T/cy-projects/yarn-v3.1.1-pnp/.yarn/__virtual__/@angular-devkit-build-angular-virtual-4aa3a8673f/packages/angular_devkit/build_angular/src/utils/webpack-browser-config.ts:158:24)
[cy:open:dev:30509]:     at async getAngularCliWebpackConfig (/Users/<USER>/cypress/npm/webpack-dev-server/dist/helpers/angularHandler.js:193:24)
[cy:open:dev:30509]:     at async angularHandler (/Users/<USER>/cypress/npm/webpack-dev-server/dist/helpers/angularHandler.js:233:27)
[cy:open:dev:30509]:     at async getPreset (/Users/<USER>/cypress/npm/webpack-dev-server/dist/devServer.js:94:20)
[cy:open:dev:30509]:     at async Function.devServer.create (/Users/<USER>/cypress/npm/webpack-dev-server/dist/devServer.js:111:61)
[cy:open:dev:30509]:     at async /Users/<USER>/cypress/npm/webpack-dev-server/dist/devServer.js:26:24
```